bpf/lib/fib.h

```diff
@@ -10,6 +10,12 @@
 #include "common.h"
 #include "l3.h"
 
+/*
+ * Translation into human language: do a fib lookup if there is more than one
+ * output device or per-endpoint routes are enabled (which => !ENABLE_ROUTING)
+ */
+#define DO_FIB_LOOKUP (!is_defined(ENABLE_SKIP_FIB) || !is_defined(ENABLE_ROUTING))
```

Comment:
I'd make it `ENABLE_FIB_LOOKUP` if possible, to be consistent with other macros.